### DISCUSSION TITLE
can we use some fancy wasm with vitejs ?

### DISCUSSION THREAD

**Discussion by sascha1337** [Budibase/budibase#1704] [Ideas]
just asking, if the stack would be ready for some Xperiments, before wasting time finding out something makes issues or sum1 already tried it

thanks, would be really fire.

**Comment by mjashanks** [comment]
In a few months, we hope to add the ability to create your own Budibase components, using Svelte. So, potentially at that point you can do this out of the box with Budibase (but I'm not too sure to be honest).

If you want to experiment now you can run Budibase from source.

1. Setup your dev environment: https://github.com/Budibase/budibase/blob/master/.github/CONTRIBUTING.md
2. Our frontend components are found in `packages/standard-components` - they use vitejs

I'd love to know what you come up with! Keep us posted :)
